examples/webgl_materials_texture3d_volume1.html

```diff
@@ -0,0 +1,308 @@
+
+	<script src="js/controls/OrthographicTrackballControls.js"></script>
+
+	<script src="js/Volume.js"></script>
+	<script src="js/VolumeSlice.js"></script>
+	<script src="js/loaders/NRRDLoader.js"></script>
+	<script src="js/shaders/VolumeShader.js"></script>
+
+	<script src="js/Detector.js"></script>
+	<script src="js/libs/stats.min.js"></script>
+	<script src="js/libs/gunzip.min.js"></script>
+	<script src="js/libs/dat.gui.min.js"></script>
+
+	<script>
+
+		if ( ! Detector.webgl ) Detector.addGetWebGLMessage();
+
+		var container,
+			stats,
+			camera,
+			controls,
+			scene,
+			renderer,
+			gui,
+			container2,
+			renderer2,
+			camera2,
+			axes2,
+			scene2;
+
+		init();
+		animate();
+
+		function init() {
+
+			// The volume renderer does not work very well with perspective yet
+			//camera = new THREE.PerspectiveCamera( 60, window.innerWidth / window.innerHeight, 0.01, 1e10 );
+			camera = new THREE.OrthographicCamera()
+			camera.position.z = 1000;
+			this.camera.right = window.innerWidth / 5.0;
+			this.camera.left = -this.camera.right;
+			this.camera.top = window.innerHeight / 5.0;
+			this.camera.bottom = -this.camera.top;
+			this.camera.near = 0.1;
+			this.camera.far = 5000;
+
+			this.camera.updateProjectionMatrix();
+
+			scene = new THREE.Scene();
+
+			scene.add( camera );
+
+			// light
+
+			var dirLight = new THREE.DirectionalLight( 0xffffff );
+			dirLight.position.set( 200, 200, 1000 ).normalize();
+
+			camera.add( dirLight );
+			camera.add( dirLight.target );
+
+			var loader = new THREE.NRRDLoader();
+			loader.load( "models/nrrd/stent.nrrd", function ( volume ) {
+
+				// Texture to hold the volume. We have scalars, so we put our data in the red channel.
+				// THREEJS will select R32F (33326) based on the RedFormat and FloatType.
+				// Also see https://www.khronos.org/registry/webgl/specs/latest/2.0/#TEXTURE_TYPES_FORMATS_FROM_DOM_ELEMENTS_TABLE
+				// TODO: look the dtype up in the volume metadata
+				var texture = new THREE.Texture3D( volume.data, volume.xLength, volume.yLength, volume.zLength );
+				texture.format = THREE.RedFormat;
+				texture.type = THREE.FloatType;
+				texture.minFilter = texture.magFilter = THREE.LinearFilter;
+				texture.unpackAlignment = 1;
+				texture.needsUpdate = true;
+
+				// Colormap textures
+				this.cmtextures = { viridis: new THREE.TextureLoader().load( 'textures/cm_viridis.png' ),
+									gray: new THREE.TextureLoader().load( 'textures/cm_gray.png' )
+								  }
+
+				// Material (shaders) to render the volume using raycasting
+				var volmaterial = new THREE.ShaderMaterial( THREE.VolumeRenderShader1 );
+				volmaterial.side = THREE.BackSide;  // The volume shader uses the backface as its "reference point"
+
+				// Apply standard volume material uniform info
+				volmaterial.uniforms.u_data = { type: 't', value: texture };
+				volmaterial.uniforms.u_size = { type: 'v3', value: [volume.xLength, volume.yLength, volume.zLength] };
```

Comment:
Kinda nit: You need extra spaces for brackets, like `func( 0, 1, 2 )` and `[ 0, 1, 2 ]`.
See the code style https://github.com/mrdoob/three.js/wiki/Mr.doob's-Code-Style%E2%84%A2

Reply:
Good point, I addressed this throughout.